liberty-maven-webapp-parent/pom.xml

```diff
@@ -50,14 +50,12 @@
                             <goals>
                                 <goal>stop-server</goal>
                             </goals>
-                            <configuration>
-                                <failOnError>false</failOnError>
-                            </configuration>
                         </execution>
                         <execution>
                             <id>create-server</id>
                             <phase>prepare-package</phase>
                             <goals>
+                                <goal>install-server</goal>
```

Comment:
If the mojo for the create-server goal implicitly calls installServerAssembly(), why does the goal need to be added here?  My guess is that it must be here explicitly to allow WDT to call the install-server goal to get the file generated?  Something still doesn't feel quite right.

Could you have the separate Mojo for generating the XML file, bind that goal to the m2e lifecycle and then have install-application call the goal implicitly?